versatileimagefield/utils.py

```diff
@@ -14,47 +15,26 @@
     VERSATILEIMAGEFIELD_FILTERED_DIRNAME
 )
 
+JPEG_MIME_TYPE = 'image/jpeg'
```

Comment:
Seems like unneeded or dead code.

Reply:
Correct, should have also removed it in https://github.com/respondcreate/django-versatileimagefield/pull/138/commits/b21c46a25c50003da75a619495f9971e8dfa68a1 where I removed my untested JPEG-fallback.